src/Form/OptionsApi.php

```diff
@@ -56,13 +58,19 @@ public function options(): array
             return $this->options;
         }
 
-        $content = @file_get_contents($this->url());
+        if (Url::isAbsolute($this->url()) === true) {
+            // URL, request via cURL
+            $data = Remote::get($this->url())->json();
+        } else {
+            // probably a local file
+            $content = @file_get_contents($this->url());
```

Comment:
It would be great, if errors weren't supressed anymore because otherwise debugging will still be very hard, see https://github.com/getkirby/kirby/issues/2330#issuecomment-559009300.

Reply:
Unfortunately `file_get_contents()` doesn't throw Exceptions, but only triggers warnings, which can only be handled with `set_error_handler()`. But that's quite a hack and therefore not really worth it. Not suppressing the warnings on the other hand will cause extra output in the API responses, which will break the Panel ("JSON could not be parsed"), so it's not useful either.

With the new implementation, most use-cases for the options API will use the `Remote` class and therefore cURL, which throws Exceptions instead that are properly displayed in the Panel. `file_get_contents()` is only kept for backwards-compatibility for use-cases where local file paths were used.

However you are right that no errors at all are bad, so I added two Exceptions manually to cover the most common error cases for local files.